x/rollup/keeper/msg_server.go

```diff
@@ -140,7 +197,7 @@ func (k *Keeper) MintETH(ctx *sdk.Context, addr sdk.AccAddress, amount sdkmath.I
 //
 // Persisted data conforms to optimism specs on L1 attributes:
 // https://github.com/ethereum-optimism/optimism/blob/develop/specs/deposits.md#l1-attributes-predeployed-contract
-func (k *Keeper) SetL1BlockInfo(ctx *sdk.Context, info derive.L1BlockInfo) error { //nolint:gocritic
+func (k *Keeper) SetL1BlockInfo(ctx sdk.Context, info derive.L1BlockInfo) error { //nolint:gocritic
```

Comment:
Purpose for this change? Not saying it isn't good, just curious

Reply:
I'd never seen a pointer to the cosmos sdk context before so I was curious if something similar was used in the cosmos sdk itself or any prominent cosmos repos. I couldn't find any references so I figured it's best to just play it safe and pass the context struct around instead of a pointer